core/arch/arm/plat-imx/conf.mk

```diff
@@ -132,6 +136,12 @@ $(call force,CFG_ARM64_core,y)
 CFG_IMX_UART ?= y
 CFG_DRAM_BASE ?= 0x40000000
 CFG_TEE_CORE_NB_CORE ?= 4
+else ifneq (,$(filter $(PLATFORM_FLAVOR),$(imx8qx-flavorlist)))
+$(call force,CFG_IMX8QX,y)
+$(call force,CFG_ARM64_core,y)
+CFG_IMX_LPUART ?= y
+CFG_DRAM_BASE ?= 0x40000000
+CFG_TEE_CORE_NB_CORE ?= 4
```

Comment:
Minor: `$(call force,CFG_TEE_CORE_NB_CORE,4)` would be more correct I suppose, since I don't expect this value can be freely chosen at compile time?
I see the same pattern is used in other places already, so feel free to keep `?=`.

Reply:
I agree, CFG_TEE_CORE_NB_CORE should be fixed. I wanted to follow the pattern already used in the file.
Let's keep things that way for this PR to match the pattern and I will fix CFG_TEE_CORE_NB_CORE for all SoC in another PR later.